whatshap/cli/split.py

```diff
@@ -172,14 +186,22 @@ def process_haplotag_list_file(
             "reads not overlapping largest phased blocks: {}".format(num_removed_reads)
         )
 
-    return readname_to_haplotype, known_reads
+    clusters = list(clusters)
+    clusters.sort()
+
+    return readname_to_haplotype, known_reads, clusters
 
 
 def _two_column_parser(line):
     cols = line.strip().split("\t")[:2]
     return cols[0], cols[1], None, None
 
 
+def _three_column_parser(line):
```

Comment:
why is there a need for a 3-column parser if the third column is always set to None?